quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -186,6 +187,7 @@ impl Default for IngestApiConfig {
             max_queue_memory_usage: Byte::from_bytes(2 * 1024 * 1024 * 1024), /* 2 GiB // TODO maybe we want more? */
             max_queue_disk_usage: Byte::from_bytes(4 * 1024 * 1024 * 1024), /* 4 GiB // TODO maybe we want more? */
             replication_factor: 1,
+            content_length_limit: 10 * 1024 * 1024 /* 10 MB */
```

Comment:
`10 << 20` would probably be better, but I think 10MB is too low that would be about 250ms worth of data 1/4 of the data a single pipeline could ingest. 100MB might be more suitable.

Reply:
Okay I will modify the default value to 100MB.

Reply:
10 was perfectly fine. Our gRPC stack does not allow messages bigger than 4 MB at the moment. Not to mention rate limiting only applies after ingest payloads are already in memory.

Reply:
I agree with @guilload as well, I have reverted it back to 10MB.